firmware/uimage.ksy

```diff
@@ -221,12 +306,24 @@ enums:
     0:
       id: none
       doc: No Compression Used
-    1: gzip
-    2: bzip2
-    3: lzma
-    4: lzo
-    5: lz4
-    6: zstd
+    1:
+      id: gzip
+      doc: gzip Compression Used
```

Comment:
IMHO it is just redundant and distracting.

Reply:
Maybe! I am not too attached to this.

Reply:
> IMHO it is just redundant and distracting.

@KOLANICH I think the same way, I had once removed these previously (https://github.com/kaitai-io/kaitai_struct_formats/commit/3d0a2a5d31004be37131b441ceb81ff2ce04a427), so this is a regression.

Reply:
> > IMHO it is just redundant and distracting.
> 
> @KOLANICH I think the same way, I had once removed these previously ([3d0a2a5](https://github.com/kaitai-io/kaitai_struct_formats/commit/3d0a2a5d31004be37131b441ceb81ff2ce04a427)), so this is a regression.

Oops. I thought I had synced properly. I will push a new commit, apologies!